test/unit/postprocessing/test_threshold_optimization.py

```diff
@@ -157,11 +154,13 @@ def test_threshold_optimization_different_input_lengths(data_X_y_sf, constraints
     inconsistent_exception_messages = {
         "inconsistent_sklearn": "Found input variables with inconsistent numbers of samples",
         "inconsistent_pandas": "All arrays must be of the same length",
+        "inconsistent_fairlearn": "X and y must have same number of rows",
     }
 
     empty_exception_messages = {
         "empty_sklearn": "Found array with 0 sample(s) (shape=(0,)) while a minimum of 1 is required.",
         "empty_pandas": "Found array with 0 sample",
+        "empty_fairlearn": "Must supply y",
```

Comment:
same here. not sure if any of the other ones have corresponding constants, would be nice to exchange them if we've missed that in the past